terraform/environments/production/variables.tf

```diff
@@ -0,0 +1,117 @@
+# NOTE
+# Variables without a default are inputted secretly. Generally only define variables with a default for non-secrets
+# ------------------------------------------------------------------
+# app
+variable "sdc_domain" {
+  description = "full domain name"
+  default     = "https://app.socialdiversity.org"
+  type        = string
+}
+
+variable "sdc_pr_domain" {
+  description = "domain of pr deploys"
+  default     = "https://social-diversity-for-children-pr-*.up.railway.app"
+  type        = string
+}
```

Comment:
Hmm I'm considering if we need this in prod. But it's probably okay.

Reply:
removed!